vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rqes/DocumentDigestEntry.kt

```diff
@@ -0,0 +1,183 @@
+@file:UseSerializers(UrlSerializer::class)
+
+package at.asitplus.wallet.lib.data.rqes
+
+import at.asitplus.KmmResult
+import at.asitplus.KmmResult.Companion.wrap
+import at.asitplus.signum.indispensable.asn1.ObjectIdSerializer
+import at.asitplus.signum.indispensable.asn1.ObjectIdentifier
+import at.asitplus.signum.indispensable.io.ByteArrayBase64Serializer
+import at.asitplus.wallet.lib.data.dif.UrlSerializer
+import io.ktor.http.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.UseSerializers
+
+@Serializable
+data class DocumentDigestEntry private constructor(
+    @SerialName("label")
+    val label: String,
+    /**
+     * base64 encoded octet representation generated using hashAlgorithmOID
+     */
+    @SerialName("hash")
+    val hash: @Serializable(ByteArrayBase64Serializer::class) ByteArray? = null,
+    @SerialName("hashAlgorithmOID")
+    val hashAlgorithmOID: @Serializable(ObjectIdSerializer::class) ObjectIdentifier? = null,
+    @SerialName("documentLocation_uri")
+    val documentLocationUri: Url? = null,
```

Comment:
Please also add documentation about all fields in the new data classes, ideally from the spec.